### PR TITLE
SNO-31-32 Create search helpers and lint search

### PR DESCRIPTION
**Pull request by kakaly**
**SNO 31 create helpers**

- Adds helpers/helper.py module

- Removes helper methods from search.py

- Resolves dependencies

- Fixes lint errors

### CODE REVIEW

**Comment by kakaly** [comment]
Depends on https://github.com/ENCODE-DCC/encoded/pull/2185

**Comment by kakaly** [comment]
@hitz - This is just a first stage refactor. Making it into a class is the ultimate goal. @caseylitton and I discussed about this and we decided to make this change one step at a time. The next step would be to refactor the helpers where ever there is scope and improve the unit testing coverage. The last step would be to integrate the ones that are more relevant into the classes that we talked about initially.

File: src/snovault/helpers/helper.py [hunk 1]
```diff
@@ -0,0 +1,567 @@
...
+        fields = {'embedded.@id', 'embedded.@type'}
+        if request.has_permission('search_audit'):
+            fields.add('audit.*')
+        types = request.registry[TYPES]
+        schemas = [types[doc_type].schema for doc_type in doc_types]
+        columns = list_visible_columns_for_schemas(request, schemas)
+        fields.update('embedded.' + column for column in columns)
+
+    # Ensure that 'audit' field is requested with _source in the ES query
+    if request.__parent__ and '/metadata/' in\
```
Comment (caseylitton) [inline src/snovault/helpers/helper.py]: revert the slash line change

**Comment by zoldello** [comment]
The code changes here (91) are subsumed by by 114- https://github.com/ENCODE-DCC/snovault/pull/114/files. Therefore, 91 is being closed in favor of 114.

**Pull request closed without merging**